modules/flowable-camel/src/test/java/org/flowable/camel/exception/CamelExceptionTest.java

```diff
@@ -95,11 +99,11 @@ public void testNonBpmnExceptionInCamel() {
         try {
             runtimeService.startProcessInstanceByKey("exceptionInRouteSynchron");
         } catch (FlowableException e) {
-            assertEquals(Exception.class, e.getCause().getClass());

```

Comment:
Why not do `assertThatThrownBy`? The `ExceptionServiceMock`  and `NoExceptionServiceMock` can be done after the `assertThatThrownBy`

Reply:
For some reason my eyes just jumped over that code block :(
Fixed.